training-server.yml

```diff
@@ -193,8 +221,19 @@
       omero.jvmcfg.percent.blitz: 50
       omero.jvmcfg.percent.indexer: 20
       omero.jvmcfg.percent.pixeldata: 20
+      omero.ldap.config: "true"
+      omero.ldap.urls: "ldap://localhost:10389"
+      omero.ldap.base: "dc=openmicroscopy,dc=org"
+      omero.ldap.group_filter: "(objectClass=groupOfUniqueNames)"
+      omero.ldap.group_mapping: "name=cn"
+      omero.ldap.new_user_group: "MyData"
+      omero.ldap.new_user_group_owner: "(owner=@{dn})"
+      omero.ldap.password: "{{ ldap_password | default(secret)}}"
```

Comment:
`'secret'` needs to be quoted, otherwise it's treated as a variable name

Reply:
Thanks, @manics. I updated https://github.com/openmicroscopy/prod-playbooks/pull/41#issuecomment-365182258